src/app/pages/admin/users/users.component.ts

```diff
@@ -0,0 +1,25 @@
+import {Component, OnInit} from '@angular/core';
+import {LittilUserService} from "../../../services/littil-user/littil-user.service";
+import {User} from "../../../api/generated";
+
+@Component({
+  selector: 'littil-admin-users',
+  templateUrl: './users.component.html',
+})
+export class UsersComponent implements OnInit {
+  users: User[] = [];
+
+  constructor(private userService: LittilUserService) { }
+
+  ngOnInit(): void {
+    this.userService.getAll().subscribe(
```

Comment:
Het kan beter zijn om de users op te halen en als observable in de template te gebruiken. Dan kan waarschijnlijk de `OnInit`ook weg. 

Dan krijg je `readonly users = this.userService.getAll().pipe(catchError(error => console.error('Error fetching users', error)))`
En in de html `{{ users | async }}`

Reply:
Users component aangepast naar async